packages/@ember/-internals/glimmer/lib/templates/empty.ts

```diff
@@ -1,4 +1,5 @@
 import { precompileTemplate } from '@ember/template-compilation';
 export default precompileTemplate('', {
   moduleName: 'packages/@ember/-internals/glimmer/lib/templates/empty.hbs',
+  strictMode: true,
```

Comment:
This one was not a problem, I just updated all of them to strict mode because why not?